source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,32 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	let deletedBranch: string | undefined;
+	const lastBranchAction = select.last('.user-select-contain > span:not(.base-ref)');
+
+	if (lastBranchAction?.closest('.TimelineItem-body')?.textContent?.includes('deleted')) {
```

Comment:
`?.` silences runtime errors. If you expect to find it, it should be `!.`

In other cases, like if `lastBranchAction` doesn’t exist, it means that nothing happened, no branch was deleted nor restored, so it should exist early with return false